convert/convert.go

```diff
@@ -120,19 +121,19 @@ func BedScoreToWig(infile string, reference map[string]chromInfo.ChromInfo) []*w
 		}
 		chromIndex = getWigChromIndex(current.Chrom, wigSlice)
 		midpoint = bedMidpoint(current)
-		if wigSlice[chromIndex].Values[midpoint].Value != 0 {
+		if wigSlice[chromIndex].Values[midpoint] != 0 {
 			log.Fatalf("Multiple scores for one position.")
 		}
 
-		wigSlice[chromIndex].Values[midpoint].Value = float64(current.Score)
+		wigSlice[chromIndex].Values[midpoint] = float64(current.Score)
 
 	}
 	return wigSlice
 }
 
 //BedScoreToWigRange uses bed entries from an input file to construct a Wig data structure where the Wig value at each position is equal to the score of an overlapping bed entry, and zero if no bed regions overlap.
```

Comment:
This function has a really similar name and description to the above function, BedScoreToWig.  Is there a difference?  I realize this is not part of the current pull request, but I am noticing it now.

Reply:
They also look identical to me. @rimangan do these look identical to you?